### PR TITLE
Updated FMEA Table and SR for Hazard Analysis

### PR DESCRIPTION
**Pull request by aaronhsli**
Updated FMEA Table and SR for Hazard Analysis

### CODE REVIEW

**Review by amoscheung99527 — APPROVED** [review]

**Review by amosyu2000 — CHANGES_REQUESTED** [review]
Don't manually update the TOC, use the VS Code extension to do it.
`Ctrl+Shift+P`
image

**Comment by aaronhsli** [comment]
Changed the formatting of the titles.

**Review by amosyu2000 — CHANGES_REQUESTED** [review]
.

File: docs/HazardAnalysis/HazardAnalysis.md [hunk 1]
```diff
@@ -76,17 +80,56 @@ The Cyclops Ride Assist system is composed of the following.
...
+| Mount housing to bicycle. | H5-1 | Mount failure. | Housing is dropped to the ground. | a. Mount is impacted by debris. <br/> b. Rattling loosens mount grip. | a. Place mount in a position unlikely to be impacted by debris (eg. not on underside of downtube) <br/> b. Instruct user to firmly tighten the mount to their bicycle. Line the gripping portion of the mount with rubber. | SR-2 <br/> IR-1 |
+
+## 8. Safety and Security Requirements
+
+### 8.1. Safety Requirements
+
+| SR1 |  A system welfare check will be conducted each time the CRA is powered on to verify that all cameras and sensors are successfully communicating with the microcontroller. |  
+|:--|:--|  
+| Rationale |A problem with the rear facing camera could result in unexpected blind spot detection behaviour (false positives or false negatives). A problem with the front or rear facing camera could result in footage of a crash being lost. A problem with crash detection sensors could result in unexpected crash detection behaviour (false positives or false negatives).|  
+|:--|:--|
```
Comment (amosyu2000) [inline docs/HazardAnalysis/HazardAnalysis.md:99]: Fix these artifacts.

**Review by amosyu2000 — APPROVED** [review]
Fix these random artifacts.
image

File: docs/HazardAnalysis/HazardAnalysis.md (hunk 1 above)
Comment (aaronhsli) [reply docs/HazardAnalysis/HazardAnalysis.md]: Reviewing to make sure it's good

**Pull request merged**